server/.golangci.yml

```diff
@@ -59,4 +59,4 @@ issues:
 
     - linters:
         - errcheck
-      path: "channels|platform|cmd/mattermost|public/model|public/plugin|public/shared/mlog|enterprise/cloud|enterprise/ldap"
```

Comment:
question, is the PR description correct with the intention of this change?

Reply:
Yes, the change is correct. It's just that the diff looks a bit off due to how GitHub renders it.